packages/webrtc-star-transport/src/utils.ts

```diff
@@ -26,10 +26,10 @@ export function cleanUrlSIO (ma: Multiaddr) {
 }
 
 export function cleanMultiaddr (maStr: string) {
-  const legacy = '/libp2p-webrtc-star'
+  const legacy = '/libp2p-webrtc-star/'
 
   if (maStr.includes(legacy)) {
```

Comment:
@achingbrain Would better fix be?
```suggestion
  if (maStr.startsWith(legacy)) {
```
I mean this works, so LGTM from me

Reply:
Could you update PR to use this instead?

Reply:
@mpetrunic I've addressed your comment. Please let me know if anymore needs to be updated.